### PR TITLE
WriteBlock and GetBlock Function Modification

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes made in this pull request:**

- Modified WriteBlock function to write gasLimit, gasUsed, transactionCount, age, uncleCount to DB
- Modified GetBlock functions to return the new fields

### CODE REVIEW

File: shyftdb/shyft_database_util.go [hunk 1]
```diff
@@ -126,7 +142,7 @@ func WriteTransactions(sqldb *sql.DB, tx *types.Transaction, blockHash common.Ha
 	gas := txData.Gas
 	data := txData.Data
 	to := txData.To
-	if(to == nil){
+	if to == nil{
```
Comment (strangepowers) [inline shyftdb/shyft_database_util.go]: should we be making style changes?

File: shyftdb/shyft_database_util.go [hunk 2]
```diff
@@ -313,11 +329,18 @@ func WriteBalanceHelper(sqldb *sql.DB, tx *types.Transaction) (SendAndReceive, s
 func GetAllBlocks(sqldb *sql.DB) string {
 	var arr blockRes
 	var blockArr string
+
```
Comment (strangepowers) [inline shyftdb/shyft_database_util.go]: new line shouldn't be here

File: shyftdb/shyft_database_util.go [hunk 3]
```diff
@@ -313,11 +329,18 @@ func WriteBalanceHelper(sqldb *sql.DB, tx *types.Transaction) (SendAndReceive, s
...
 			number,
 			hash,
-			coinbase
+			coinbase,
+			gasused,
+			gaslimit,
+			txcount,
+			unclecount,
+			age
+
```
Comment (strangepowers) [inline shyftdb/shyft_database_util.go]: another new line

File: shyftdb/shyft_database_util.go [hunk 4]
```diff
@@ -355,12 +394,27 @@ func GetBlock(sqldb *sql.DB) string {
...
 	var hash string
 	var coinbase string
-	row.Scan(&num, &hash, &coinbase)
+	var gasUsed string
+	var gasLimit string
+	var txCount string
+	var uncleCount string
+	var age string
+
+	row.Scan(&num, &hash, &coinbase, &gasUsed,
```
Comment (strangepowers) [inline shyftdb/shyft_database_util.go]: keep style consistent, so on different lines

File: shyftdb/shyft_database_util.go (hunk 1 above)
Comment (GregTheGreek) [reply shyftdb/shyft_database_util.go]: Are you referring to assigning the variables before using them in line 106?

File: shyftdb/shyft_database_util.go (hunk 4 above)
Comment (GregTheGreek) [reply shyftdb/shyft_database_util.go]: agreed

**Pull request merged**